File: TP1 - C++/data/models/time_sheet.cpp
#include "../../domain/entities/time_sheet.hpp"

TimeSheet::TimeSheet(){};

TimeSheet::TimeSheet(string id, vector<Point > points):
    id(id),
    points(points){};

TimeSheet::~TimeSheet(){}

string TimeSheet::getId(){
    return this->id;
}

vector <Point> TimeSheet::getPoints(){
    return this->points;
}

Point TimeSheet::getPoint(int index){
    return this->points[index];
}

void TimeSheet::setPoints(vector<Point> points){
    this->points = points;
}

File: TP2 - Java/src/domain/entities/TimeSheet.java
package domain.entities;

import java.util.ArrayList;
import java.util.List;

public class TimeSheet {
  private final String id;
  private List<Point> points;

  public TimeSheet(String id, List<Point> points) {
    this.id = id;
    this.points = new ArrayList<Point>();
  }

  public String getId() {
    return id;
  }

  public List<Point> getPoints() {
    return points;
  }

  public void setPoints(List<Point> points) {
    this.points = points;
  }
}
